applications/GeoMechanicsApplication/tests/prescribed_derivative_tests/prescribed_acceleration/MaterialParameters.json

```diff
@@ -0,0 +1,29 @@
+{
+  "properties": [
+    {
+      "model_part_name": "PorousDomain.Soil_two_phase",
+      "properties_id": 1,
+      "Material": {
+        "constitutive_law": {
+          "name": "GeoLinearElasticPlaneStrain2DLaw"
+        },
+        "Variables": {
+          "IGNORE_UNDRAINED": false,
+          "YOUNG_MODULUS": 150000,
+          "POISSON_RATIO": 0.3,
+          "DENSITY_SOLID": 2.0e3,
+          "DENSITY_WATER": 1.0e3,
+          "POROSITY": 0.3,
+          "BULK_MODULUS_SOLID": 1.0e12,
+          "BULK_MODULUS_FLUID": 2.0e9,
+          "PERMEABILITY_XX": 1.0E-10,
+          "PERMEABILITY_YY": 1.0E-10,
+          "PERMEABILITY_XY": 0.0,
+          "DYNAMIC_VISCOSITY": 1.0e-6,
+          "THICKNESS": 1.0
```

Comment:
I'm not sure if we might be able to delete a couple of these, do you know @WPK4FEM?